app/pages/sections/commencement-date/template.test.js

```diff
@@ -0,0 +1,106 @@
+import { componentTester } from '../../../test-utils/componentTester';
+import manifest from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/sections/commencement-date/template.njk',
+  },
+};
+
+const context = {
+  ...manifest,
+  title: 'Commencement date for order-id',
+  backLinkHref: '/organisation/order-1',
+  csrfToken: 'mockCsrfToken',
+};
+
+describe('commencement-date page', () => {
+  it('should render a backLink', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const backLink = $('[data-test-id="go-back-link"]');
+      expect(backLink.length).toEqual(1);
+      expect(backLink.text().trim()).toEqual('Go back');
+      expect($(backLink).find('a').attr('href')).toEqual('/organisation/order-1');
+    });
+  }));
+
+  it('should render the page title', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const title = $('h1[data-test-id="commencement-date-page-title"]');
+      expect(title.length).toEqual(1);
+      expect(title.text().trim()).toEqual(context.title);
+    });
+  }));
+
+  it('should render the page description', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const description = $('h2[data-test-id="commencement-date-page-description"]');
+      expect(description.length).toEqual(1);
+      expect(description.text().trim()).toEqual(context.description);
+    });
+  }));
+
+  describe('form fields', () => {
+    xit('should render hidden input with csrf token', componentTester(setup, (harness) => {
```

Comment:
`x` to remove

Reply:
I will remove when I add the token to the form - I left this here to remind me to add the test